src/components/Feeds/BusinessFeed.js

```diff
@@ -26,10 +26,11 @@ function BusinessFeed({ businesses, onSearch, selectedFilters }) {
           />
           <SimpleGrid columns={[null, 1, 2]} spacing={10}>
             {businesses.map(business => {
+              console.log('HEY', business);
```

Comment:
Probs should remove this :)

Reply:
oops just removed as you pinged me. It's early 🌞